src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,83 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int RESIZE_VALUE = 2;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] data = new Node[16];
```

Comment:
let's move initialization on the constructor and create a new const with arrays length